_data/tools.yml

```diff
@@ -616,6 +616,17 @@
   v2: false
   v3: true
 
+- name: openapi-spring-webflux-validator
+  category:
+    - miscellaneous
+    - schema-validators
+    - data-validators
+  github: https://github.com/cdimascio/openapi-spring-webflux-validator
+  language: Java/Kotlin
+  description: 🌱 A friendly kotlin library to validate API endpoints using an OpenApi 3.0 and Swagger 2.0 specification
```

Comment:
Swagger 2.0 is OpenAPI 2.0

Reply:
oops. thanks for the heads up.
fixed in this PR https://github.com/apisyouwonthate/openapi.tools/pull/147